app/models/user.rb

```diff
@@ -343,6 +343,27 @@ def perform_export_photos!
     update exporting_photos: false
   end
 
+  ######### Data import ##################
+  mount_uploader :import, JsonUploader
+
+  def data_import(importedfile)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for data_import is too high. [51.36/20]